### PR TITLE
Week1-AmirHossein

### PR DESCRIPTION
**Pull request by amirhshad**

### CODE REVIEW

File: Week1/js-exercises/ex5-catWalk.js [hunk 1]
```diff
@@ -9,5 +9,47 @@
...
+
+const imgCat = document.getElementsByTagName('img')[0];
+imgCat.style.left = '0px';
+let speed = 1;
+
+const winScreen = window.innerWidth-imgCat.width;
+
+const catMoves = setInterval(catWalk, 5)
+
+function catWalk() {
```
Comment (n4cr) [inline Week1/js-exercises/ex5-catWalk.js:23]: Does this work even when you have defined the function `catWalk` after using it?

**Comment by n4cr** [comment]
You Branch has conflicts. Please resolve the conflicts. First merge the master of `samravan/Javascript2` into your repo, resolve the conflicts and then push again

**Comment by n4cr** [comment]
This seems like an old PR. I'll close it

File: Week1/js-exercises/ex5-catWalk.js (hunk 1 above)
Comment (amirhshad) [reply Week1/js-exercises/ex5-catWalk.js:23]: I haven't finished this exercise yet! Actually I'm stuck in this one!

**Pull request closed without merging**